### ISSUE TITLE
Port to Mac OS X

### ISSUE THREAD

**Issue by gsemet** [Guake/guake#246]
port seems to have python-vte

trying with the home-brew environment which seems more mature.

Ref: http://apple.stackexchange.com/questions/88742/where-can-i-get-python-vte-bindings-for-the-mac-os-x

**Comment by varemenos** [comment]
Would some extra :+1:s help raise this issue's priority?

**Comment by Stibbons** [comment]
on mac I use Total Terminal. Guake is quite a bit hard to port. All these gtk stuff is not meant to work on mac. The globalhotkeys should be completely reworked to use mac os hotkey system....

**Comment by varemenos** [comment]
That's rather sad to hear (I would really to use guake everywhere if possible) but it's certainly understandable.

**Comment by sivaramsk** [comment]
A newbie to MAC and a huge guake fan - a bit disappointed to see it is unportable. One thing that I could not make it work in mac with iterm2 or TotalTerminal is drop down to come up in a single hotkey (not a combination). In total terminal, could not make a it full screen with a hotkey like F11 in guake - got to press Expand horizontally multiple times which is annoying - any way I could make it to full screen with single key combination?? Guess guake is the champion and we got ti compromise at office with iterm2 or totalterminal.

**Comment by Stibbons** [comment]
Yes it work. I have it directly with the [^2] key on the French keyboard with total terminal on Mac, like guake on Linux

**Comment by sivaramsk** [comment]
You are talking about full screen?? I don't see anyoption in total terminal to make it to full screen, would be great if you would tall me how

**Comment by Stibbons** [comment]
http://totalterminal.binaryage.com/

it is a sort of plugin on the mac terminal. You can set this terminal to be full screen by default and call it with this magic hotkey.

**Comment by sivaramsk** [comment]
This I know. What I need is - is there any way that I can make the TotalTerminal to full screen in a single keypress. Now you have press Alt + Command + Down arrow several times to make it to full screen.

**Comment by Stibbons** [comment]
I don't have my Mac here. But it exists :)

**Comment by zhangyubaka** [comment]
Get a ImportError: No module named gconf even have gconf installed,no idea...

**Comment by brunocechet** [comment]
how is this awesome port ?

**Comment by zx1986** [comment]
I am using iTerm2 to get this: https://stackoverflow.com/questions/30850430/iterm2-hide-show-like-guake
